### ISSUE TITLE
Improve quarry texture

### ISSUE THREAD

**Issue by deathcap** [voxel/voxel-quarry#3]
Currently it uses `texture: 'furnace_top'`, but a real quarry block texture would be nice. Various texture packs have these textures but would want to add it to [ProgrammerArt](https://github.com/deathcap/ProgrammerArt) as a default fallback.

**Comment by z3t0** [comment]
@deathcap I can work on this, also is there any correlation between the format of minecraft texture packs and ProgrammerArt?

**Comment by deathcap** [comment]
@z3t0 Yes the ProgrammerArt pack is distributed in a format compatible with Minecraft 1.6+ "resource packs", also packaged in various other formats in https://github.com/deathcap/ProgrammerArt/tree/master/scripts. Also relevant: https://github.com/deathcap/ProgrammerArt/issues/13
